src/Microsoft.Health.Dicom.Core/Features/Query/QueryService.cs

```diff
@@ -44,7 +54,16 @@ public class QueryService : IQueryService
 
             ValidateRequestIdentifiers(message);
 
-            QueryExpression queryExpression = _queryParser.Parse(message);
+            IEnumerable<CustomTagEntry> customTags = await _customTagStore.GetCustomTagsAsync(null, cancellationToken);
+
+            Dictionary<QueryResource, HashSet<DicomTag>> queryResourceToCustomTagMapping = null;
```

Comment:
nit: not setting queryResourceToCustomTagMapping  to null, instead, returns what from GenerateQueryResourceToCustomTagMapping which is empty dictionary. it could simplify the code little bit.